qutebrowser/completion/models/listcategory.py

```diff
@@ -63,6 +63,9 @@ def set_pattern(self, val):
         val = re.sub(r' +', r' ', val)  # See #1919
         val = re.escape(val)
         val = val.replace(r'\ ', '.*')
+        if len(val) > 5000:  # avoid crash on huge search terms (#5973)
+            log.completion.warning("Pattern too long ({})".format(len(val)))
```

Comment:
You can use f-strings nowadays:

```suggestion
            log.completion.warning(f"Pattern too long ({len(val)})")
```